assignments/objects.js

```diff
@@ -17,28 +17,79 @@ const example = {
   "gender": "F"
 }
 
-// Write your intern objects here:
+// Write your intern objects here: 
+const Intern1 = {
```

Comment:
Excellent work creating the objects! Note that the quotes around the keys are not required